File: GuerrillaAPP/Domain/Recurso.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GuerrillaAPP.Domain
{
    public class Recurso
    {
        public Recurso(int idRecurso,int cantidadRecurso,String nombreRecurso){
            this.idRecurso = idRecurso;
            this.cantidadRecurso = cantidadRecurso;
            this.nombreRecurso = nombreRecurso;

        }
        public int idRecurso { get; set; }
        public int cantidadRecurso { get; set; }
        public String nombreRecurso { get; set; }
    }

}

File: GuerrillaApp/src/main/java/domain/Recurso.java
package domain;

public class Recurso {
	private int idRecurso;
	private int cantidadRecurso;
	private String nombreRecurso;

	public Recurso(int idRecurso, int cantidadRecurso, String nombreRecurso) {
		super();
		setIdRecurso(idRecurso);
		setNombreRecurso(nombreRecurso);
		setCantidadRecurso(cantidadRecurso);
	}

	public int getIdRecurso() {
		return idRecurso;
	}

	public void setIdRecurso(int idRecurso) {
		this.idRecurso = idRecurso;
	}

	public int getCantidadRecurso() {
		return cantidadRecurso;
	}

	public void setCantidadRecurso(int cantidadRecurso) {
		this.cantidadRecurso = cantidadRecurso;
	}

	public String getNombreRecurso() {
		return nombreRecurso;
	}

	public void setNombreRecurso(String nombreRecurso) {
		this.nombreRecurso = nombreRecurso;
	}

	@Override
	public String toString() {
		return "Recurso [idRecurso=" + idRecurso + ", cantidadRecurso=" + cantidadRecurso + ", nombreRecurso="
				+ nombreRecurso + "]";
	}

}
